modules/ROOT/pages/nom-user-management/index.adoc

```diff
@@ -16,6 +17,9 @@ The effective access levels of a NOM user are reflected in the NOM UI appropriat
 |Administrator
 |Has access to features of NOM as configured by NOM Admin.
 Can not administer NOM users.
+
+|read-only Administrator (for SSO-user only)
+| Has view-only access to the following features of NOM: DBMS status, metrics and logs, as well as agents and the upgrades page.
```

Comment:
Is anything excluded from this role? E.g security is not mentioned. If not, maybe simpler to state "Has view access to all features and DBMSs"